packages/askui-nodejs/src/execution/dsl.ts

```diff
@@ -825,7 +411,7 @@ export class FluentFilters extends FluentBase {
    *     .customElement({
    *         customImage: './logo.png', // required
    *         name: 'myLogo', // optional
-   *         threshold: 0.5, // optional, defaults to 0.5
+   *         threshold: 0.9, // optional, defaults to 0.9
```

Comment:
I think you've reverted the default parameter from @adi-wan-askui .

Reply:
Yes, indeed.